storage/CMakeLists.txt

```diff
@@ -43,6 +41,7 @@ set(
 )
 
 omim_add_library(${PROJECT_NAME} ${SRC})
+omim_link_libraries(${PROJECT_NAME} jansson)
```

Comment:
I don't like the idea of linking the storage lib (as opposed to, say, storage_tests executable) with jansson.
Has the old way stopped working? Why?

Reply:
Like I said below, jansson needs configuration, only include dependencies, like before will cause missing generated files. Library is 100k only, could you please motivate the problem?

Reply:
We were only linking binaries before so if we are to make a change here, it should be made carefully.

Reply:
Please format according to other omim_link_libraries calls (e.g. https://github.com/mapsme/omim/blob/master/storage/storage_tests/CMakeLists.txt)